installation.html

```diff
@@ -11,312 +11,216 @@
+        like <a href="http://www.sourcetreeapp.com/" target="_blank">SourceTree</a> if you are starting with Git.
+    </li>
+    <li>Install Node.js from <a href="http://nodejs.org/" target="_blank">the Node.js website</a>. This will also
+        install <code>npm</code>, which is the node package manager we are using in the next commands.
+    </li>
     <li>Install Yeoman: <code>npm install -g yo</code></li>
     <li>Install Bower: <code>npm install -g bower</code></li>
-    <li>Depending on your preferences, install either <a href="http://gruntjs.com/" target="_blank">Grunt</a> (recommended) with <code>npm install -g grunt-cli</code> or <a href="http://gulpjs.com/" target="_blank">Gulp.js</a> with <code>npm install -g gulp</code>.</li>
+    <li>Depending on your preferences, install either <a href="http://gruntjs.com/" target="_blank">Grunt</a>
+        (recommended) with <code>npm install -g grunt-cli</code> or <a href="http://gulpjs.com/"
+                                                                       target="_blank">Gulp.js</a> with <code>npm
+            install -g gulp</code>.
+    </li>
     <li>Install JHipster: <code>npm install -g generator-jhipster</code></li>
 </ol>
 
 <p>To find more information, tips and help, please have a look at
-<a href="http://yeoman.io/learning/index.html" target="_blank">the Yeoman "getting starting" guide</a> and at the <a href="https://docs.npmjs.com/" target="_blank">NPM documentation</a> before
-<a href="https://github.com/jhipster/generator-jhipster/issues?state=open" target="_blank">submitting a bug</a>.</p>
+    <a href="http://yeoman.io/learning/index.html" target="_blank">the Yeoman "getting starting" guide</a> and at the <a
+            href="https://docs.npmjs.com/" target="_blank">NPM documentation</a> before
+    <a href="https://github.com/jhipster/generator-jhipster/issues?state=open" target="_blank">submitting a bug</a>.</p>
 
 <h2>Docker installation (for advanced users)</h2>
 
 <h3>Information</h3>
 
 <p>
-JHipster has a specific <a href="https://github.com/jhipster/jhipster-docker" target="_blank">jhipster-docker project</a>, which provides a <a href="https://www.docker.io/" target="_blank">Docker</a> container.
+    JHipster has a specific <a href="https://github.com/jhipster/jhipster-docker" target="_blank">jhipster-docker
+    project</a>, which provides a <a href="https://www.docker.io/" target="_blank">Docker</a> container.
 </p>
 <p>
-This project makes a Docker "Trusted build" that is available on:
+    This project makes a Docker "Trusted build" that is available on:
 </p>
 <p>
-<a href="https://registry.hub.docker.com/u/jdubois/jhipster-docker/" target="_blank">https://registry.hub.docker.com/u/jdubois/jhipster-docker/</a>
+    <a href="https://registry.hub.docker.com/u/jdubois/jhipster-docker/" target="_blank">https://registry.hub.docker.com/u/jdubois/jhipster-docker/</a>
```

Comment:
The official link has changed (with the new hub) :
https://hub.docker.com/r/jdubois/jhipster-docker/